Fix flaky test

This was an error from a test failure during an unrelated CI run:

```
=================================== FAILURES ===================================
_________________________ test_check_throttle_handler __________________________

    def test_check_throttle_handler() -> None:
        @get("/path1")
        def handler1() -> None:
            return None
    
        @get("/path2")
        def handler2() -> None:
            return None
    
        def check_throttle_handler(request: Request[Any, Any]) -> bool:
            return request.url.path == "/path1"
    
        config = RateLimitConfig(rate_limit=("second", 1), check_throttle_handler=check_throttle_handler)
    
        with create_test_client(route_handlers=[handler1, handler2], middleware=[config.middleware]) as client:
            response = client.get("/path1")
            assert response.status_code == HTTP_200_OK
    
            response = client.get("/path1")
>           assert response.status_code == HTTP_429_TOO_MANY_REQUESTS
E           assert 200 == 429
E            +  where 200 = <Response [200]>.status_code

tests/middleware/test_rate_limit.py:110: AssertionError

=========================== short test summary info ============================
FAILED tests/middleware/test_rate_limit.py::test_check_throttle_handler - ass...
================== 1 failed, 732 passed, 3 warnings in 30.90s ==================
```

The error seems to be transient, as re-run of the CI job passed.

--- a/tests/middleware/test_rate_limit.py
+++ b/tests/middleware/test_rate_limit.py
@@ -100,7 +100,7 @@ def test_check_throttle_handler() -> None:
     def check_throttle_handler(request: Request[Any, Any]) -> bool:
         return request.url.path == "/path1"
 
-    config = RateLimitConfig(rate_limit=("second", 1), check_throttle_handler=check_throttle_handler)
+    config = RateLimitConfig(rate_limit=("minute", 1), check_throttle_handler=check_throttle_handler)
 
     with create_test_client(route_handlers=[handler1, handler2], middleware=[config.middleware]) as client:
         response = client.get("/path1")

